lib/android/src/main/java/com/airbnb/android/react/maps/AirMapModule.java

```diff
@@ -142,12 +142,14 @@ public void onSnapshotReady(@Nullable Bitmap snapshot) {
 
   @ReactMethod
   public void pointForCoordinate(final int tag, ReadableMap coordinate, final Promise promise) {
+    final ReactApplicationContext context = getReactApplicationContext();
+    double density = (double) context.getResources().getDisplayMetrics().density;
```

Comment:
I applied these changes to my local node module and was unable to build for android. 

I believe this line needs read `final double density = (double) [context.getResources().getDisplayMetrics().density;`

Once I made this change to the local file, I was able to build. I can also confirm that the PR's code does fix the issue described for android.